pymatgen/io/openmm/simulations.py

```diff
@@ -0,0 +1,100 @@
+"""
+This functionality eventually belongs in Atomate, but for the sake of rapidly developing
+a OpenMM IO, I will leave it here for now.
+
+The idea is that each function will operate on a openmm.Simulation and propagate the
+simulation forward in time.
+"""
+
+# base python
+from typing import Union, List
+
+# scipy
+import numpy as np
+
+# openmm
+from openmm import MonteCarloBarostat
+from openmm.unit import kelvin, atmosphere
+from openmm.app import Simulation
+
+
+__author__ = "Orion Cohen, Ryan Kingsbury"
+__version__ = "1.0"
+__maintainer__ = "Orion Cohen"
+__email__ = "orion@lbl.gov"
+__date__ = "Nov 2021"
+
+
+def equilibrate_pressure(
+    simulation: Simulation,
+    steps: int,
+    temperature: float = 298,
+    pressure: float = 1,
+):
+    """
+    Equilibrate the pressure of a simulation in the NPT ensemble.
+
+    Adds and then removes a openmm.MonteCarlo Barostat to shift the system
+    into the NPT ensemble.
+
+    Parameters
+    ----------
+    simulation : the openmm.Simulation to propagate.
+    steps : the length of the heating, holding, and cooling stages. Steps is number of steps.
```

Comment:
Can you please change `the length of the heating, holding, and cooling stages. Steps is number of steps.` to `the length of the pressure equilibration. Steps is number of steps.` ?

Reply:
Done!